pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -304,6 +304,39 @@ func createTargetConfigController(ctx context.Context, syncCtx factory.SyncConte
 		return true, err
 	}
 
+	// update the condition if external cloud provider has been applied
+	externalCloudProviderApplied, err := isExternalCloudProviderApplied(operatorSpec)
+	if err != nil {
+		return true, err
+	}
+	if externalCloudProviderApplied {
+		cond := operatorv1.OperatorCondition{
+			Type:   "CloudControllerOwner",
+			Status: operatorv1.ConditionFalse,
+		}
+		if _, _, err := v1helpers.UpdateStaticPodStatus(c.operatorClient, v1helpers.UpdateStaticPodConditionFn(cond)); err != nil {
+			return true, err
+		}
+	}
```

Comment:
Does this wait until the changes have rolled out before applying the condition?

Reply:
I added a check to make sure that static pods are available before setting the condition.